src/rust/en.omegascans/src/parser.rs

```diff
@@ -0,0 +1,213 @@
+use aidoku::{
+	error::Result, helpers::uri::encode_uri_component, prelude::format, std::net::HttpMethod,
+	std::net::Request, std::String, std::Vec, Chapter, Filter, FilterType, Manga,
+	MangaContentRating, MangaPageResult, MangaStatus, MangaViewer, Page,
+};
+
+const BASE_API_URL: &str = "https://api.omegascans.org";
```

Comment:
this can be imported from `lib.rs` instead (I think `crate::BASE_API_URL` would work).